packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,279 @@
+  const elevationBlur = elevation && elevation >= 1 ? (elevation / 16) * 0.1 + 0.3 : 0;
+  return `
+    width: fit-content;
+    transition: filter ${timings.slow};
+    filter: drop-shadow(0rem ${elevationYOffset}rem ${elevationBlur}rem rgba(0,0,0,${0.6 - elevation * 0.1}));
+  `;
+}}
+  
+`;
+const ValueContainer = styled(ButtonContainer)`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  
+  line-height: initial;
+  width: 15rem;
+  padding: 0.5rem;
+`;
+const ValueIconContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  
+  width: 3rem;
+`;
+
+const ValueItem = styled.div`
+  width: 100%;
+  text-align: left;
+`;
+
+const OptionsContainer = styled.div`
+  background: white;
+  position: absolute;
+  max-height: 10rem;
+  overflow-y: scroll;
+  width: 15rem;
+  border: 0.5px solid ${colors.blueCharcoal25};
+`;
+
+const OptionItem = styled.div`
+  height: 2rem;
+  display: flex;
+  align-items: center;
+  
+  font-family: Montserrat,Roboto,sans-serif;
+  
+  &:focus,
+  &:hover {
+    background: ${colors.blueCharcoal50};
+    cursor: pointer;
+    outline: none;
+  }
+`;
+const CheckContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  
+  padding-right: 0.2rem;
+  width: 2rem;
+`;
+
+export interface DropdownProps {
+  StyledContainer?: string & StyledComponentBase<any, {}>;
+  StyledValueContainer?: string & StyledComponentBase<any, {}>;
+  StyledValueItem?: string & StyledComponentBase<any, {}>;
+  StyledOptionsContainer?: string & StyledComponentBase<any, {}>;
+  StyledOptionItem?: string & StyledComponentBase<any, {}>;
+
+  clearable?: boolean,
+  elevation?: number,
+  multi?: boolean,
+  name: string,
+  onClear?: () => void,
+  onSelect?: (selected: string | Array<string>) => void,
+  options: Array<string>,
+  tabIndex?: number,
+  values?: Array<string>,
+}
+
+// TODO Placeholder text -- Wait until input is finalized
+export const Dropdown = ({
+  StyledContainer = Container,
+  StyledValueContainer = ValueContainer,
+  StyledValueItem = ValueItem,
+  StyledOptionsContainer = OptionsContainer,
+  StyledOptionItem = OptionItem,
+
+  clearable = false,
+  elevation = 0,
+  multi = false,
+  name,
+  onClear,
+  onSelect,
+  options,
+  tabIndex = 0,
+  values = [],
+}: DropdownProps) => {
+  const [state, setState] = useState<{ isOpen: boolean, selectedValues: Array<string>, id: string}>({
+    isOpen: false,
+    selectedValues: values,
+    id: name,
+  });
+
+  const handleBlur = useCallback((e: React.FocusEvent) => {
+    e.preventDefault();
+    const target = e.nativeEvent.relatedTarget as HTMLElement | null;
+    // check if we're focusing on something we don't control
+    if (!target || (target.id && !target.id.startsWith(state.id))) {
+      setState(curState => ({ ...curState, isOpen: false }));
+    }
+  }, [state.id]);
```

Comment:
We should probably add an onBlur callback to go with this guy

Reply:
Good point.

Reply:
Done.